packages/postgres-database/src/simcore_postgres_database/migration/versions/cd53975abda2_refactoring_of_resource_tracker_.py

```diff
@@ -0,0 +1,94 @@
+"""refactoring of resource_tracker_container table
+
+Revision ID: cd53975abda2
+Revises: a8762d5d43ae
+Create Date: 2023-07-10 17:15:34.126749+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "cd53975abda2"
+down_revision = "a8762d5d43ae"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    type = postgresql.ENUM("DYNAMIC_SIDECAR", "USER_SERVICE", name="containertype")
+    type.create(op.get_bind())
+
+    op.execute("DELETE FROM resource_tracker_container;")
+
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "resource_tracker_container",
+        sa.Column("cpu_limit", sa.Numeric(precision=3, scale=2), nullable=False),
+    )
+    op.add_column(
+        "resource_tracker_container",
+        sa.Column("memory_limit", sa.BigInteger(), nullable=False),
```

Comment:
Minor: Should it maybe be nullable, just from a code-design perspective? If there is no memory limit, I guess it should be set to `NULL` instead of 0, which means 0MB. Maybe this is not a topic for this PR though

Reply:
for now, I would like to keep it not nullable -> these numbers are the ones that the user specifies from the front-end and we will need to use them in the pricing (so I want it to fail quickly if there is some problem, but if we will need the null value in the future we will add it)